src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -18,10 +18,7 @@
     :skip-step? (fn [db] (some? (get-in db [:wallet :ui :send :recipient])))}
    {:screen-id  :screen/wallet.select-asset
     :skip-step? (fn [db] (or (token-selected? db) (collectible-selected? db)))}
-   {:screen-id  :screen/wallet.send-input-amount
-    :skip-step? (fn [db]
-                  (or (not (token-selected? db))
-                      (some? (get-in db [:wallet :ui :send :amount]))))}
+   {:screen-id :screen/wallet.send-input-amount}
```

Comment:
I think `input-amount` screen should never be skipped

Reply:
yeah seems right to me too!

Reply:
When we want to send a collectible instead of a regular token, we don't navigate to `:screen/wallet.send-input-amount`. For collectibles with balance greater than 1 (could be ERC-1155 or community tokens), we have `:screen/wallet.select-collectible-amount`. Am I missing something?

Reply:
Nope, you're correct Brian! 
I forgot about that

Reply:
Thanks @briansztamfater! Updated it to skip when `tx-type` is collectible

Reply:
We don't want that either, because there are multiple collectibles.
But in the case of multiple collectibles the screen is different

Reply:
@J-Son89 @briansztamfater is there a case where we need to navigate to `:screen/wallet.send-input-amount` when sending a collectible?